recipes/snpsift/meta.yaml

```diff
@@ -17,6 +17,10 @@ build:
   noarch: generic
   number: 0
   skip: False
+  run_exports:
```

Comment:
This does not look correct. You should pin here future snpsift version. So you indicate when the API breaks.
```
- {{ pin_compatible('snpsift', max_pin="x.x") }}
```

Reply:
you mean like this
```suggestion
  run_exports:
    - {{ pin_compatible('snpsift', max_pin="x.x") }}
    - {{ pin_compatible('python', max_pin="3.9.*") }}
    - {{ pin_compatible('perl', max_pin="5.32.*") }}
    - {{ pin_compatible('openjdk', max_pin="11.*") }}
```
or like this?
```suggestion
  run_exports:
    - {{ pin_compatible('snpsift', max_pin="x.x") }}
```

Reply:
Ok, I understand now